docs/uptime/analytics/open-web-analytics.md

```diff
@@ -0,0 +1,150 @@
+
+    systemctl enable mariadb
+    systemctl enable httpd
+
+4. CentOS 6.5 users run:
+
+    chkconfig mysql on
+    chkconfig httpd on
+
+##2. Set up MySQL
+
+2. Then (if you already have mysql installed you can skip this step):
+
+    mysql_secure_installation
+
+Answer yes to all questions -- make sure to set a strong password for root (press enter when prompted for current root password since it is initially blank). This is the root user for mysql and is not related the systems root user.
+
+3. Next run the following. Enter your mysql root password when asked:
+ 
+    mysql -u root -p
+
+4. The following SQL commands will create a database named `owadb` with a user named `owadbuser` and with the password `owadbpassword`. Be sure to change the password to something better of your own choice. This information will be needed later to configure OWA.
+
+    CREATE DATABASE owadb;
+    GRANT ALL PRIVILEGES ON owadb.* TO owadbuser@localhost IDENTIFIED BY 'owadbpassword';
+    FLUSH PRIVILEGES;
+    quit
+
+5. Now you need to restart your Linode:
+
+    reboot
+
+##3. Install OWA
+
+1. Once your Linode comes back up, navigate to your document root folder for your webserver, usually `/var/html/www`:
+
+    cd /var/www/html
+
+2. Now download the OWA package:
+
+    wget https://github.com/padams/Open-Web-Analytics/archive/1.5.7.tar.gz
+
+{: .note}
+>
+>The number 1.5.7 is the current version and may be different by the time you read this. Please check http://www.openwebanalytics.com/ for the latest information.
+
+3. Unpack it:
+
+    tar xf 1.5.7.tar.gz
+
+4. Delete tar file (you may want to wait until everything is finished and working before you do this just in case you need to a fresh version of the OWA files):
+
+    rm -rf 1.5.7.tar.gz
+
+5. Change ownership of the `owa` folder to apache (assuming you are running Apache, if not change it to your httpd’s user).
+
+    chown -R apache:apache Open*
+
+6. Rename the OWA folder to something reasonable:
+
+    mv Open-Web-Analytics-1.5.7 owa
+
+##4. Configure OWA
+
+1. Now navigate to the installation in your webbrowser. (Replace `your.domain` with the domain of your Linode):
+
+    http://your.domain/owa/install.php
+{: .note}
+>
+>Your server must be configured with a fully qualified domain name and not just an IP address.
+
+2. You should see a configuration page for your OWA installation. The first field will be filled in by OWA with the path to your OWA installation. The other fields on the page should be filled in as follows:
+ 
+	Database Host: localhost
+
+	Database Name: owadb
+
+	Database User: owadbuser
+
+	Database Password: owadbpassword
+
+Of course, `owadbpassword` should be the password with which you created your SQL database in step 2.4.
+
+![Screen shot of OWA set up screen.](/docs/assets/OWA.png)
+
+3. Click "Continue..." and create a user account on the next page. This user account will be how you log in to see your OWA statistics.
```

Comment:
During my tech testing it appears that this step in the installation page also has us create the first site to track, so that when we log in we already have some code presented. This affects both this step and the next section.